### PR TITLE
REL-4014: correct GHOST IFU size

### PR DESCRIPTION
**Pull request by swalker2m**
Corrects the GHOST standard resolution IFU size.  In theory. The requirements are difficult to parse.

### CODE REVIEW

File: bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala [hunk 1]
```diff
@@ -142,34 +142,33 @@ final class TpeGhostIfuFeature extends TpeImageFeature("GHOST", "Show the patrol
...
     }
 
     g2d.setColor(TpeGhostIfuFeature.IfuColor)
-    val hex: Area = TpeGhostIfuFeature.regularHexagon().createTransformedArea(trans)
-    g2d.fill(hex)
+    g2d.fill(TpeGhostIfuFeature.standardResolutionIfuArcsec.createTransformedArea(trans))
   }
 
   /**
-   * Draw the SR sky fibers relative to the center of SRIFU1. Thery are extremely small.
```
Comment (sraaphorst) [inline bundle/jsky.app.ot/src/main/scala/jsky/app/ot/tpe/feat/TpeGhostIFUFeature.scala:158]: Ooops.

**Review by sraaphorst — APPROVED** [review]
Apart from the not sure if reading correctly, this looks fine and we may as well test.

**Review by tpolecat — COMMENTED** [review]
Well at least doing Java2D is kind of fun. How do we verify this against the actual instrument (since the spec is hard to interpret)?

**Comment by swalker2m** [comment]
> How do we verify this against the actual instrument (since the spec is hard to interpret)?

I'm hoping Steve will help with that.  Presumably they can compare vs an actual image, but I expect Steve will be able to spot some issues just by looking at the TPE.  I don't have much confidence that this is correct, but I am confident that it is closer to correct than it was before so I'll await the next iteration.

**Pull request merged**